api/src/main/java/uk/gov/dhsc/htbhf/claimant/reporting/payload/ReportPropertiesFactory.java

```diff
@@ -95,11 +100,13 @@ protected Long calculateQueueTime(LocalDateTime timestamp) {
 
     protected Map<String, Object> createCommonCustomMetrics(ReportEventMessageContext context) {
         Map<String, Object> customMetrics = new TreeMap<>();
-        Claimant claimant = context.getClaim().getClaimant();
+        Claim claim = context.getClaim();
+        Claimant claimant = claim.getClaimant();
         LocalDate atDate = context.getTimestamp().toLocalDate();
         customMetrics.put(CLAIMANT_AGE.getFieldName(), Period.between(claimant.getDateOfBirth(), atDate).getYears());
-        long childrenUnder4 = getNumberOfChildrenUnderFour(context.getDatesOfBirthOfChildren(), context.getTimestamp().toLocalDate());
-        long childrenUnder1 = getNumberOfChildrenUnderOne(context.getDatesOfBirthOfChildren(), context.getTimestamp().toLocalDate());
+        List<LocalDate> dobOfChildrenUnder4 = claim.getCurrentIdentityAndEligibilityResponse().getDobOfChildrenUnder4();
+        long childrenUnder4 = getNumberOfChildrenUnderFour(dobOfChildrenUnder4, context.getTimestamp().toLocalDate());
+        long childrenUnder1 = getNumberOfChildrenUnderOne(dobOfChildrenUnder4, context.getTimestamp().toLocalDate());
```

Comment:
You're calling `context.getTimestamp().toLocalDate()` in both these lines, that could be refactored out.

Reply:
true, though that was already the case